index.js

```diff
@@ -38,12 +41,15 @@ function Scroller(scroller, content, render, isPrepend, isSticky, cb) {
       var m = queue.shift()
       var r = render(m)
       append(scroller, content, r, isPrepend, isSticky)
+      obv.set(queue.length)
     }
   }
 
   function scroll (ev) {
-    if(isEnd(scroller, buffer, isPrepend) || !isFilled(content))
+    if(isEnd(scroller, buffer, isPrepend) || !isFilled(content)) {
       pause.resume()
+      add()
```

Comment:
I previously removed this - I think it was related to the massive addition of messages. 
I thought `pause.resume()` was  sufficient to get `add()` called, as the resume restarts the stream, which ultimately leads to addition (if the appropriate conditions are met).